net/sscep/Makefile

```
@@ -0,0 +1,53 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sscep
+PKG_VERSION:=0.10.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=v${PKG_VERSION}.tar.gz
+PKG_SOURCE_URL:=https://github.com/certnanny/sscep/archive/
+PKG_HASH:=489cc8e093986776eb3f15082bf766778f707176f3cd604bf0ef1008da06b8e5
+
+PKG_MAINTAINER:=John Marrett <johnf@zioncluster.ca>
+PKG_LICENSE:=
```

```suggestion
PKG_LICENSE:=BSD-3-Clause
```

from the COPYING file, looks like it's BSD 3-Clause

---

@BKPepe was right (see https://github.com/openwrt/packages/pull/20683#discussion_r1140684101), the file contains multiple licenses.

I think it would be most appropriate to list all of the licenses (space-separated) in `PKG_LICENSE`.

---

Ok, that makes sense, I'll do that.

---

I wasn't able to add the OpenOSP license, I've put in the two that are defined in SPDX.